Meta/CMake/flatpak/org.ladybird.Ladybird.desktop

```diff
@@ -0,0 +1,32 @@
+[Desktop Entry]
+Name=Ladybird
```

Comment:
@bugaevc Do you want any co-authorship or anything on this commit/patch series/etc? These freedesktop files are lifted from your ladybird-gtk project.

Reply:
It would make sense to mention that in the commit message, either `Co-authored-by`, or at least reference the fact that these files originated in ladybird-gtk4. The translations were also contributed by various people.

By the way, putting this into `flatpak/` sounds wrong, there's nothing Flatpak-specific about a `.desktop` file. I hope you're also installing it when building normally, outside of Flatpak?

Reply:
> I hope you're also installing it when building normally, outside of Flatpak?

I noticed this during my once-over last night as well, and made a note to rename the command to ENABLE_INSTALL_FREEDESKTOP_FILES or some such. I'm not 100% sure on *when* to install these files? `if (LINUX AND NOT ANDROID)` ? `if (UNIX AND NOT APPLE AND NOT ANDROID)` ? having a cmake option makes that not my problem to figure out :D